main.tex

```diff
@@ -306,35 +269,45 @@ \subsection*{\textit{Vgsc} non-synonymous nucleotide variation}
 %
-In \textit{Aedes aegypti} the \texttt{V402L} mutation has been found alone and in conjunction with a mutation just one residue away from \texttt{I1527T}, in this species these mutations, alone and in combination, were shown to strongly reduce VGSC sensitivity to pyrethroids in \textit{Xenopus} oocytes.
+We also found that the two variant alleles affecting codon 402, both of which induce a \texttt{V402L} substitution, were in strong linkage with \texttt{I1527T} ($D' \geq 0.8$; Figure \ref{fig:ld}), and almost all haplotypes carrying \texttt{I1527T} also carried a \texttt{V402L} substitution.
 %
-The \texttt{V402L} substitution has also been independently associated with pyrethroid resistance in bedbugs \cite{Yoon2008}.
+The most parsimonious explanation for this pattern of linkage is that the \texttt{I1527T} mutation occurred first, and mutations in codon 402 subsequently arose on this genetic background or recombined with it.
 %
-Other substitutions at this codon have also been associated with resistance, \texttt{V402A/G} in the moth crop pests \emph{Helicoverpa zea} \cite{Hopkins2010} and \texttt{V402M} in \emph{Heliothis virescens}, the latter of which has been shown experimentally to confer resistance in \textit{Xenopus} oocytes \cite{Park1997, Lee2013}.
+Substitutions in codon 402 have been found in a number of other insect species and shown experimentally to confer pyrethroid resistance \cite{Dong2014}. 
 %
 Because of the limited geographical distribution of these alleles, we hypothesize that the \texttt{I1527T+V402L} combination represents a pyrethroid resistance allele that arose in West African \textit{An. coluzzii} populations.
 %
-However, the \texttt{L995F} allele is at higher frequency (85\%) in our Burkina Faso \textit{An. coluzzii} population, and is known to be increasing in frequency \cite{Toe2014}, therefore \texttt{L995F} may provide a stronger resistance phenotype and is thus replacing \texttt{I1527T+V402L}.
+However, the \texttt{L995F} allele is at higher frequency (85\%) in our Burkina Faso \textit{An. coluzzii} population, and is known to be increasing in frequency \cite{Toe2014}, therefore \texttt{L995F} may provide a stronger resistance phenotype and is replacing \texttt{I1527T+V402L}.
 %%
 
 
 %% Other novel variants.
 %
-The remaining 4 novel alleles (two separate nucleotide substitutions causing \texttt{M490I}; \texttt{A1125V}; \texttt{V1254I}) do not occur in combination with any known resistance allele.
+The remaining 4 novel alleles (two separate nucleotide substitutions causing \texttt{M490I}; \texttt{A1125V}; \texttt{V1254I}) did not occur in combination with any known resistance allele.
 %
 All are private to a single population, and (to our knowledge) none have previously been found in other species.
-%
-All occur within internal linker domains of the protein and are therefore unlikely to directly cause resistance, discussed further below.
```

Comment:
Removed this as I think we've learned substitutions pretty much anywhere in the protein can cause resistance!